app/views/layouts/application.html.erb

```diff
@@ -47,6 +47,7 @@
         <div class="main-menu">
           <ul id="side-main-menu" class="side-menu list-unstyled">
             <li class="<%= @all_leads_active %>"><a href="/"> <i class="icon-home"></i><span>All Leads</span></a></li>
+            <li class="<%= @new_lead_active %>"><a href="/leads/new"> <i class="icon-flask"></i><span>New Lead</span></a></li>
```

Comment:
I think you're missing the initialization of `@new_lead_active`

Reply:
Hi Dan.  Thanks for the reminder.  We made this change.  Could you approve it?  Thank you!